docs/rules/web-matrix.md

```diff
@@ -1,50 +1,51 @@
-## Rules support matrix by techno
+# Rules support matrix by techno
 
 Each rule need to be develop for each technology to be available in SonarQube plugin.
 
 Here is the list of rules already available in ecoCode project code.
```

Comment:
I think that the two sentences above are not relevant: they make us think that it is possible to integrate the rules in all languages and that they are all already available for use. Maybe we can take the opportunity to review the wording?

Reply:
@glalloue ? your opinion ?

Reply:
I pushed my suggested modification, feel free to comment or reject ;)

Reply:
Yes, better! Thank you :+1: